lib/xpi/utils.js

```diff
@@ -6,6 +6,8 @@
 var join = require("path").join;
 var when = require("when");
 var fs = require("fs-promise");
+var Zip = require("jszip");
```

Comment:
ooh, can we use `jszip` for other things? I tried to fix another issue [like this](https://github.com/mozilla-jetpack/jpm/pull/410) but it broke tests in mysterious ways. Maybe I'll try to resurrect it with `jszip`.